projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -157,133 +300,146 @@ export function NewsletterAccess( {
-									<Dropdown
-										placement="bottom-end"
-										contentClassName="edit-post-post-visibility__dialog"
-										focusOnMount
-										renderToggle={ ( { isOpen, onToggle } ) => (
-											<Button
-												variant="tertiary"
-												onClick={ onToggle }
-												aria-expanded={ isOpen }
-												aria-label={ sprintf(
-													// translators: %s: Current newsletter post access.
-													__( 'Select audience: %s', 'jetpack' ),
-													accessLabel
-												) }
-											>
-												{ accessLabel }
-											</Button>
-										) }
-										renderContent={ ( { onClose } ) => (
-											<div className="editor-post-visibility">
-												<InspectorPopoverHeader
-													title={ __( 'Audience', 'jetpack' ) }
-													help={ __( 'Control how this newsletter is viewed.', 'jetpack' ) }
-													onClose={ onClose }
-												/>
-												<NewsletterAccessChoices
-													accessLevel={ accessLevel }
-													socialFollowers={ socialFollowers }
-													emailSubscribers={ emailSubscribers }
-													paidSubscribers={ paidSubscribers }
-													onChange={ setPostMeta }
-												/>
-											</div>
-										) }
-									/>
-								</FlexBlock>
-							) }
+							{ /* Display the uneditable access level when the user doesn't have edit privileges*/ }
+							{ ! canEdit && <span>{ accessLabel }</span> }
+						</Flex>
+
+						<NewsletterNotice
+							accessLevel={ _accessLevel }
+							socialFollowers={ socialFollowers }
+							emailSubscribers={ emailSubscribers }
+							paidSubscribers={ paidSubscribers }
+							showMisconfigurationWarning={ showMisconfigurationWarning }
+						/>
+
+						<FlexBlock>
+							<NewsletterLearnMore />
+						</FlexBlock>
+					</Flex>
+				</PanelRow>
+			) }
+		/>
+	);
+}
+
+export function NewsletterAccessPrePublishSettings( {
+	accessLevel,
+	setPostMeta,
+	socialFollowers,
+	emailSubscribers,
+	paidSubscribers,
+	showMisconfigurationWarning,
+} ) {
+	const { hasNewsletterPlans, stripeConnectUrl, isLoading } = useSelect( select => {
+		const { getProducts, getConnectUrl, isApiStateLoading } = select(
+			'jetpack/membership-products'
+		);
+
+		return {
+			isLoading: isApiStateLoading(),
+			stripeConnectUrl: getConnectUrl(),
+			hasNewsletterPlans:
+				getProducts()?.filter( product => product.subscribe_as_site_subscriber )?.length !== 0,
+		};
+	} );
+
+	if ( isLoading ) {
+		return (
+			<Flex direction="column" align="center">
+				<Spinner />
+			</Flex>
+		);
+	}
 
-							{ ! showVisibilityRestrictedMessage && ! withModal && canEdit && (
+	const _accessLevel = accessLevel ?? accessOptions.everybody.key;
+	const accessLabel = accessOptions[ _accessLevel ]?.label;
+
+	return (
+		<PostVisibilityCheck
+			render={ ( { canEdit } ) => (
+				<PanelRow className="edit-post-post-visibility">
+					<Flex direction="column">
+						{ showMisconfigurationWarning && MisconfigurationWarning() }
```

Comment:
Should we render `MisconfigurationWarning` as a React component verses calling it as function here?

Reply:
Good point. I missed that.

Reply:
Yes, and it [this is how it is right now](https://github.com/Automattic/jetpack/pull/30446/files#diff-eba2301ede604e2450a0b3ea7d13cdc88d427f67d0f1777028aa9e2e154623c3R318-R319). You are looking at an outdated version (not sure how you got there).